task4выделениеметодов/task4выделениеметодов/Program.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task4выделениеметодов
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+
+            Console.WriteLine(RoundedResult());
+
+            Console.ReadKey();
+        }
+
+        private static double RoundedResult()
+        {
+            return Math.Round(SumOfSin(2, 3) / SumOfCos(2, 3) + (SumOfCos(5, 7)) / SumOfSin(5, 7), 3);
+        }
+
+        static double SumOfCos(int x, int y)
+        {
+            return (Math.Cos(x) + Math.Cos(y));
+        }
+
+        static double SumOfSin(int x, int y)
+        {
+
+            return sin(x) + sin(y);
+
+        }
```

Comment:
Результат не надо округлять до трех знаков после запятой. В задании сказано "вывести с точностью до трех знаков после запятой". То есть вычислить его нужно максимально точно, а при выводе на консоль использовать соответсвующее форматирование.
Основная задача этого задания - выделить метод, то есть в заданной формуле увидеть повторяющееся выражение (одно, а не два!) и оформить его вычисление в виде отдельного метода.
То есть ваш код должен быть в таком стиле
```suggestion
        private static double GetResult()
        {
            return CalculateExpression(2, 3) + CalculateExpression(5, 7);
        }

        static double CalculateExpression(double x, double y)
        {
            //здесь код вычисления значения повторяющегося выражения
        }
```